spec/models/list_spec.rb

```diff
@@ -42,4 +42,12 @@ def create_list_with_books(*books)
       end
     end
   end
+
+  it 'sets a slug when saved do' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.